pkg/distributor/ha_tracker_test.go

```diff
@@ -897,3 +897,36 @@ func getSumOfHistogramSampleCount(families []*dto.MetricFamily, metricName strin
 
 	return sum
 }
+
+// Test that values are synced to KVStore on starting
+func TestStartingSync(t *testing.T) {
+	cluster := "c1"
+	replica := "r1"
+
+	codec := GetReplicaDescCodec()
+	kvStore, closer := consul.NewInMemoryClient(codec, log.NewNopLogger(), nil)
+	t.Cleanup(func() { assert.NoError(t, closer.Close()) })
+
+	mock := kv.PrefixClient(kvStore, "prefix")
+	c, err := newHATracker(HATrackerConfig{
+		EnableHATracker:        true,
+		KVStore:                kv.Config{Mock: mock},
+		UpdateTimeout:          15 * time.Second,
```

Comment:
The update timeout is usually at least 2x smaller than the failover timeout. Is there a special reason for keeping it __larger__ than the timeout?